@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -440,10 +440,12 @@ export default class Esxi extends EventEmitter {
         </soapenv:Envelope>`,
     })
     const text = await res.text()
-    const matches = text.match(/<FetchResponse[^>]*>(.*)<\/FetchResponse>/)
-
+    const matches = text.match(/<FetchResponse[^>]*>(.*)<\/FetchResponse>/s)
```

Comment:
You can use [`@vates/xml`](https://github.com/vatesfr/xen-orchestra/tree/master/%40vates/xml) if you want instead.

Reply:
I do want to use it, but it will probably also change the result,so I think I wont add it in a patch release
Hopefully I will be using it by then end of the month